bbc/simorgh
# Use psammead-paragraph

Resolves #997

This PR uses `@bbc/psammead-paragraph`
Updated snapshots.

**Testing instructions**
```
git checkout use-psammead-paragraph
npm ci
npm run dev
```
Visit http://localhost:7080/news/articles/c9rpqy7pmypo

Compare with https://www.test.bbc.com/news/articles/c9rpqy7pmypo

The paragraphs should be the same.
